### ISSUE TITLE
NullRefException when executing two test assemblies with given --domain=Multiple explicitly

### ISSUE THREAD

**Issue by GraueEminenz79** [nunit/nunit-console#356]
Hi,
I ran into a NullReferenceException when executing two different test assemblies (and a subset of test cases filtered by where) and using the `--domain=Multiple` parameter, like
`nunit3-console.exe A.dll B.dll --where:class=TestFixtureFromA OR class=TestFixtureFromB --domain=Multiple`
somehow fails during summary report gerneration:
```
at NUnit.ConsoleRunner.ColorConsoleWriter.WriteLabel(String label, Object option, ColorStyle valueStyle)
   at NUnit.ConsoleRunner.ColorConsoleWriter.WriteLabelLine(String label, Object option, ColorStyle valueStyle)
   at NUnit.ConsoleRunner.ResultReporter.WriteSummaryReport()
   at NUnit.ConsoleRunner.ConsoleRunner.RunTests(TestPackage package, TestFilter filter)
   at NUnit.ConsoleRunner.Program.Main(String[] args)
```
Hint: `--noc` does not help to workaround the issue...but omitting the `--domain=Multiple` and using the default behaviour for more than one test assembly will throw **NO** exception.

**Comment by ChrisMaddock** [comment]
@GraueEminenz79 I think this is a duplicate of https://github.com/nunit/nunit-console/issues/29

There's some workarounds in that issue - would you try adding `--inprocess` or `--process=Separate` depending on your preference, and see if that solves you issue for now?

**Comment by GraueEminenz79** [comment]
@ChrisMaddock It does! Thanks very much. Sorry for not looking closer. I will track the issue for myself by #29

**Comment by ChrisMaddock** [comment]
No problem - thanks for reporting it anyway. (Also - we'd welcome a Pull Request to fix #29 - if you were interested in digging deeper! 😄 Happy to help point in the right direction.)

**Comment by ChrisMaddock** [comment]
(Although I wonder actually if this may be fixed by #351 - which does related things...)

**Comment by CharliePoole** [comment]
I don't see how that command-line can work as shown above. The phrase `--where:class=TestFixtureFromA` would be taken as an option, `OR` would be another option and `class=TestFixtureFromB` would be a third. Did you actually use quotation marks? If so, where?
